### ISSUE TITLE
go-xcat should not print out the "Congratulations" message on update, only on install

### ISSUE THREAD

**Issue by whowutwut** [xcat2/xcat-core#1642]
When running `go-xcat update` , there's no need to print the message the end that xCAT has been installed correctly because we are updating xCAT and not installing xCAT for the first time. 

```
...
...
Updated:
  perl-xCAT.noarch 4:2.12.2-snap201608040616                                    
  xCAT.ppc64le 0:2.12.2-snap201608040616                                        
  xCAT-buildkit.noarch 4:2.12.2-snap201608040616                                
  xCAT-client.noarch 4:2.12.2-snap201608040616                                  
  xCAT-genesis-base-ppc64.noarch 1:2.12-snap201607221057                        
  xCAT-genesis-scripts-ppc64.noarch 1:2.12.2-snap201608040616                   
  xCAT-server.noarch 4:2.12.2-snap201608040616                                  

Complete!
Command "repoquery" not found.
go-xcat: Above listed required command(s) not found.
Command "repoquery" not found.
go-xcat: Above listed required command(s) not found.

xCAT Core Packages
==================

Package Name                Installed                      In Repository                 
------------                ---------                      -------------                 
Command "repoquery" not found.
go-xcat: Above listed required command(s) not found.
perl-xCAT                   2.12.2-snap201608040616                                      
xCAT                        2.12.2-snap201608040616                                      
xCAT-buildkit               2.12.2-snap201608040616                                      
xCAT-client                 2.12.2-snap201608040616                                      
xCAT-confluent              (not installed)                                              
xCAT-genesis-scripts-ppc64  2.12.2-snap201608040616                                      
xCAT-genesis-scripts-x86_64 (not installed)                                              
xCAT-server                 2.12.2-snap201608040616                                      
xCATsn                      (not installed)                                              
xCAT-SoftLayer              (not installed)                                              
xCAT-test                   (not installed)                                              
xCAT-vlan                   (not installed)                                              

xCAT Dependency Packages
========================

Package Name                Installed                      In Repository                 
------------                ---------                      -------------                 


Congratulations
===============

The fact that you got this far is a strong indication that xCAT bas been
installed correctly.

Please notice if this is the first time you install xCAT. You need to do one
of the following.

1. Log out and then log in again, or
2. run the following command to set the environment variables.

    for sh,
        `source /etc/profile.d/xcat.sh`
    or csh,
        `source /etc/profile.d/xcat.csh`
Command "repoquery" not found.
go-xcat: Above listed required command(s) not found.
```

Other comments
- Why do we need to list out xCAT Core Package and xCAT Dependency Packages at the end...  the output from yum seems sufficient to identify the packages that have been updated... .  By listing (not installed) will this cause a **NEW** user of xCAT to think something went wrong?

**Comment by neo954** [comment]
Good point.

**Comment by whowutwut** [comment]
@neo954  @daniceexi  I tested go-xcat today and it does not contain the changes.   Have we thought about how to best sync go-xcat?     I don't like the idea of having to manually upload a copy.  If we sync with xcat.org repo, we still need to git pull.....  which is also a manual step.

**Comment by daniceexi** [comment]
@whowutwut I understand your concern. If we'd like to make all the update to be automatic, we may do following:

1. put go-xcat in xcat.org repo
2. dev should check in go-xcat to both master and stable version
3. on xcat.org, run a cron job to pull new changes periodically

**Comment by whowutwut** [comment]
@daniceexi  for 2, maybe we should only check into master and create pull requests to the stable branch when we want to update..    Instead of having the developers check in to two branches... 

Is there a need to keep `go-xcat` in the tools directory? If the user did not use go-xcat to install, would they want to use it to update?   It will be more difficult but not impossible to sync go-xcat using the build scripts...

**Comment by daniceexi** [comment]
@whowutwut I am fine with your idea. And if we move go-xcat to xcat.org repo, I think we should remove it from xcat-core since it's very hard to sync it between two repos.

**Comment by whowutwut** [comment]
Yea, we should just update the version manually for now... perhaps there will be less changes are time goes by...

**Comment by daniceexi** [comment]
Yes, I agree. We should not put effort to automate the thing which does not happen often. 
The owner of the ``go-xcat`` defect should verify it base on the one in xcat.org, this can make sure the new version of ``go-xcat`` is updated to xcat.org.

**Comment by whowutwut** [comment]
Fixed with `go-xcat` version 1.0.2
